delta/sourcing-psql/src/main/scala/ch/epfl/bluebrain/nexus/delta/sourcing/state/ScopedStateGet.scala

```diff
@@ -0,0 +1,18 @@
+package ch.epfl.bluebrain.nexus.delta.sourcing.state
+
+import ch.epfl.bluebrain.nexus.delta.sourcing.model.Tag.Latest
+import ch.epfl.bluebrain.nexus.delta.sourcing.model.{EntityType, ProjectRef, Tag}
+import doobie.implicits._
+import doobie.{ConnectionIO, Get, Put}
+
+object ScopedStateGet {
+
+  def apply[Id: Put, S: Get](tpe: EntityType, project: ProjectRef, id: Id, tag: Tag): ConnectionIO[Option[S]] =
```

Comment:
Allow to fetch a state without having a dependency on the whole project module.
Meant to be extended to other cases to break more circular dependencies

Reply:
Same suggestion as `GlobalStateGet`, could it be a trait?